recipes/bwa2/2.0pre2_npg/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "2.0pre2_npg" %}
+
+package:
+  name: bwa-aligner-2
```

Comment:
Name should be consistent with https://github.com/kjsanger/npg_conda/blob/conda-recipe-structure/README.md i.e. `bwa-mem2`